Explorer/Assets/Scripts/SceneRunner/ECSWorld/ECSWorldFacade.cs

```diff
@@ -46,6 +51,10 @@ public void Dispose()
 
             Profiler.EndSample();
 
+#if UNITY_EDITOR
+            SystemGroupSnapshot.Instance.Unregister(this.sceneShortInfo,systemGroupWorld);
```

Comment:
Do you really need to pass the name for `Unregister`? Can't it be handled by `SystemGroupWorld` only?